pkg/util/net/ip/ip.go

```diff
@@ -83,3 +83,17 @@ func isIPv6Disabled(filename string) bool {
 
 	return data[0] == '1'
 }
+
+func IsIPv4Address(address string, useSloppyParsing bool) bool {
```

Comment:
Ahh.. we are back a few steps now.

- Using a boolean as an argument is always a smell, and here it is obvious. This function does two things and the caller controls it. So there is no reason to have one function.. there should be two. But if there are two, you do not really need it, right?
- I prefer no one will use util/net/ip package now that we have the network package available. We need to move everything out of here.. or at least the ones we care about.